tests/unit/test_configuration.py

```diff
@@ -135,7 +146,7 @@ def test_get_configuration_with_user_override(mocker):
             },
         }
     }
-    _ = get_configuration(config)
+    _ = get_configuration(config)  # type: ignore [arg-type]
```

Comment:
huh, was this a revealed failure or something?

Reply:
Yes, and I ignored it because I thought it was something that will go away when we define our config object more directly.